Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Events/InteractableOnPressReceiver.cs

```diff
@@ -11,33 +12,71 @@ namespace Microsoft.MixedReality.Toolkit.UI
     /// </summary>
     public class InteractableOnPressReceiver : ReceiverBase
     {
+
         [InspectorField(Type = InspectorField.FieldTypes.Event, Label = "On Release", Tooltip = "The button is released")]
         public UnityEvent OnRelease = new UnityEvent();
 
+        public enum PressType
+        {
+            NearAndFar = 0,
+            NearOnly = 1,
+            FarOnly = 2
+        }
+
+        [InspectorField(Label = "Press Interaction Type", Tooltip = "Specify whether press event is for near or far interaction", Type = InspectorField.FieldTypes.DropdownInt, Options = new string[] { "Near and Far", "Near Only", "Far Only" })]
+        public int PressTypeFilter = (int)PressType.NearAndFar;
```

Comment:
@Troy-Ferrell and @Railboy  berni + i couldn't find a way to actually use an enum for the public field, it looks like the custom inspector only supports int lists. Do you have ideas / suggestions for how to use an enum here?